### DISCUSSION TITLE
MagickNET breaking changes 7.17 > 7.20

### DISCUSSION THREAD

**Discussion by ghost** [ImageMagick/ImageMagick#2198] [Help] [answered]
Hello,

Is this the correct place for questions about MagickNET ? the existing forum room is closed.
If not, please give me the correct link.

**Comment by dlemstra** [comment, marked as answer]
The question should be asked here instead: https://github.com/dlemstra/Magick.NET/discussions. Going to wait for you to post it there before responding.
